src/test/java/tests/Reqres.java

```diff
@@ -0,0 +1,183 @@
+
+@Data
+public class Reqres {
+    public static final String URL = "https://reqres.in";
+    public static final String LIST_USERS = "/api/users?page=2";
+    public static final String SINGLE_USER = "/api/users/2";
+    public static final String SINGLE_USER_NOT_FOUND = "/api/users/23";
+    public static final String LIST_RESOURCE = "/api/unknown";
+    public static final String SINGLE_RESOURCE = "/api/unknown/2";
+    public static final String SINGLE_RESOURCE_404 = "/api/unknown/23";
+    public static final String CREATE = "/api/users";
+    public static final String PUT = "/api/users/2";
+    public static final String REGISTER_SUCCESSFUL = "/api/register";
+    public static final String LOGIN = "/api/login";
+    public static final String DELAYED = "/api/users?delay=3";
+    public String name = "morpheus";
+    public String job = "leader";
+    public String jobUpdate = "zion resident";
+
+    @Test
+    public void listUserIdTest() {
+        String body = given()
+                .when()
+                .get(URL + LIST_USERS)
+                .then()
+                .extract().body().asString();
+        UsersList listUser = new Gson().fromJson(body, UsersList.class);
+        int id = listUser.getData().get(1).getId();
+        Assert.assertEquals(id, 8);
+    }
+
+    @Test
+    public void singleUserResponseTest() {
+        given()
+                .when()
+                .get(URL + SINGLE_USER)
+                .then()
+                .statusCode(200);
+    }
+
+    @Test
+    public void singleUserNotFoundError() {
+        given()
+                .when()
+                .get(URL + SINGLE_USER_NOT_FOUND)
+                .then()
+                .statusCode(404);
+    }
+
+    @Test
+    public void listResourceTest() {
+        String body = given()
+                .when()
+                .get(URL + LIST_RESOURCE)
+                .then()
+                .extract().body().asString();
+        ListResource res = new Gson().fromJson(body, ListResource.class);
+        String pantoneValue = res.getData().get(4).getPantoneValue();
+        Assert.assertEquals(pantoneValue, "17-1456");
+    }
+
+    @Test
+    public void singleResourceTest() {
+        String body = given()
+                .when()
+                .get(URL + SINGLE_RESOURCE)
+                .then()
+                .extract().body().asString();
+        SingleResource name = new Gson().fromJson(body, SingleResource.class);
+        String singleName = name.getData().getName();
+        Assert.assertEquals(singleName, "fuchsia rose");
+    }
+
+    @Test
+    public void singleResourceNotFoundTest() {
+        new ReqresAdapter().get(SINGLE_RESOURCE_404, 404);
+    }
+
+    @Test
+    public void createPostTest() {
+        Suite suite = Suite.builder()
+                .name(name)
+                .job(job)
+                .build();
+        String actualResult = new ReqresAdapter().post(CREATE, "{\n" +
```

Comment:
your 'suite' object is not used here. Put suite object and convert to json via gson here instead of hardcoding JSON request payload.

Example:
Gson converter = new Gson();
String actualResult = new ReqresAdapter().post(CREATE, converter.toJson(suite) , 201).getBody().jsonPath().getJsonObject("name");

Reply:
fix